src/ECER.Managers.Registry/CommunicationMapper.cs

```diff
@@ -0,0 +1,23 @@
+﻿using AutoMapper;
+using ECER.Resources.Accounts.Communications; 
+
+namespace ECER.Managers.Registry
+{
+  internal class CommunicationMapper : Profile
+  {
+
+    public CommunicationMapper()
+    {
+      CreateMap<Communication, ECER.Managers.Registry.Contract.Communications.Communication>()
+           .ForMember(dest => dest.Id, opt => opt.MapFrom(src => src.Id ?? string.Empty))
+           .ForMember(dest => dest.Text, opt => opt.MapFrom(src => src.Body))
+           .ReverseMap()
```

Comment:
No reversal is needed, and the map should be flipped

Reply:
I followed the application as a reference. in the ApplicationHandler we convert Resources to Contract.Applications. In the communications we need that convert as well.